Library/Tracking/KoalaTests.swift

```diff
@@ -25,51 +25,62 @@ final class KoalaTests: TestCase {
     let device = MockDevice(userInterfaceIdiom: .phone)
     let screen = MockScreen()
     let koala = Koala(
-      bundle: bundle, client: client, config: config, device: device, loggedInUser: nil,
-      screen: screen
+      bundle: bundle,
+      client: client,
+      config: config,
+      device: device,
+      loggedInUser: nil,
+      screen: screen,
+      distinctId: "abc-123"
     )
 
     koala.trackAppOpen()
-    XCTAssertEqual(["App Open", "Opened App"], client.events)
 
     let properties = client.properties.last
 
-    XCTAssertEqual("Apple", properties?["manufacturer"] as? String)
-
-    XCTAssertEqual(bundle.infoDictionary?["CFBundleVersion"] as? Int, properties?["app_version"] as? Int)
-    XCTAssertEqual(
-      bundle.infoDictionary?["CFBundleShortVersionString"] as? String,
-      properties?["app_release"] as? String
-    )
-    XCTAssertNotNil(properties?["model"])
-    XCTAssertEqual(device.systemName, properties?["os"] as? String)
-    XCTAssertEqual(device.systemVersion, properties?["os_version"] as? String)
-    XCTAssertEqual(UInt(screen.bounds.width), properties?["screen_width"] as? UInt)
-    XCTAssertEqual(UInt(screen.bounds.height), properties?["screen_height"] as? UInt)
+    // FIXME: The following properties cannot be reliably tested
```

Comment:
Best to keep the `FIXME:` here or track in a Jira ticket?

Reply:
It's in both, I can remove it from here if you think it's too noisy.